docs/spec_draft.md

```diff
@@ -1337,6 +1338,39 @@ defined and one of the following:
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.is_finite
+
+### Semantics
+
+Performs element-wise check whether the value in `x` is finite (i.e. is neither
++inf, -inf, nor NaN) and produces a `y` tensor.
```

Comment:
Additionally, let's also add a second sentence: "Implements the `isFinite` operation from the IEEE-754 specification".